conftest.py

```diff
@@ -43,6 +43,9 @@ def check_required_loopback_interfaces_available():
 
 
 def pytest_addoption(parser):
+    parser.addoption("--default-sstable-format", action="store", default="bti",
```

Comment:
I am not sure if we need `--default` maybe `--sstable-format` is enough?
up to you.